ftw/simplelayout/tests/builders.py

```diff
@@ -11,6 +13,12 @@ class ContenPageBuilder(DexterityBuilder):
 class TextBlockBuilder(DexterityBuilder):
     portal_type = 'ftw.simplelayout.TextBlock'
 
+    def with_dummy_image(self):
+        image = Path(__file__).joinpath('..', 'assets', 'fullhd.jpg').abspath()
```

Comment:
Well, I need an image with a huge dimension... ok at least with a 400px width. I chose an image, which is large enough for all plone default scales.